src/App.tsx

```diff
@@ -1,61 +1,36 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { AddTodoForm } from './components/AddTodoForm';
+
+import { Todo } from './types/Todo';
+
+import { getUserById } from './utils/getUserById';
```

Comment:
[CHECKLIST ITEM 5] Don't use `import` keyword in non-root components for data that can be passed down from the App component. Ensure that all necessary data is prepared and passed down from the App component to avoid direct imports in child components.